gsa/src/gmp/models/nvt.js

```diff
@@ -149,8 +149,11 @@ class Nvt extends Info {
 
     if (isDefined(ret.severities)) {
       const {severity} = ret.severities;
-      ret.severity = parseSeverity(severity.score / 10);
-      ret.severityOrigin = parseText(severity.origin);
+      ret.severity = parseSeverity(severity?.score / 10);
+      ret.severityOrigin = parseText(severity?.origin);
+      ret.severityDate = isDefined(severity?.date)
+        ? parseDate(severity.date)
+        : undefined;
```

Comment:
the function already checks for undefined
```suggestion
      ret.severityDate = parseDate(severity.date);
```